upgrading/automated_upgrades.adoc

```diff
@@ -756,6 +797,34 @@ You can set the `openshift_logging_elasticsearch_replace_configmap` parameter to
 ConfigMap can cause the upgrade to fail. The default is set to `false`. For more information, see the parameter 
 in xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[specify logging Ansible variables].
 
+. Scale down your Fluentd and Elasticsearch pods to stop data ingestion and ensure the cluster state does not change.
```

Comment:
No replicas for damonset controller pods. Is 'scale down' the proper word ?  shall we use deschuler the damonset pods by either modify the nodeSelector in ds/logging-fluentd or update the node label.

Reply:
Are we expected the user to these steps prior openshift-logging playbooks?

Reply:
@jcantrill @lukas-vlcek Please comment on @anpingli question on scale down. 

Anping, the scale down/descheduling is done before running playbooks.

Reply:
Anping, the scale down/descheduling is done before running playbooks.

Reply:
@mburke5678  We need to split this sentence to two steps. 1) de-scheduler fluentd 2) scale down elasticsearch.   and the step 'scale down elasticsearch' should be after step 7.
@jcantrill  @lukas-vlcek I think we need to explain the 'scale down Elasticsearch' .  I think that means 'set dc/elasticsearchs replicas number=0' rather than elasicsearch cluster 'scale down'

Reply:
It is not possible to 'scale down' a daemonset.  Essentially you have to modify the node selector such that it won't mache any nodes.

@anpingli correct, we will need to set each ES DS replica to 0